ui/src/components/mobileservices/BoundServiceRow.test.js

```diff
@@ -8,12 +8,14 @@ import BoundServiceRow from './BoundServiceRow';
 describe('BoundServiceRow', () => {
```

Comment:
@ziccardi @wei-lee 
Here's my removed test for UPS case:
```
describe('BoundServiceRow - UPS - 1 binding', () => {
  const service = {
    getConfiguration: () => undefined,
    getConfigurationExt: () => `[
      {"type":"android","typeLabel":"Android","url":"https://ups-mdc.127.0.0.1.nip.io/#/app/8936dead-7552-4b55-905c-926752c759af/variants/2d76d1eb-65ef-471c-8d21-75f80c3f370f","id":"2d76d1eb-65ef-471c-8d21-75f80c3f370f"}
    ]`,
    getDocumentationUrl: () => undefined,
    getBindingName: () => 'test-ups-r66b9',
    getIconClass: () => 'fa fa-something',
    getId: () => 'UPS',
    getLogoUrl: () => undefined,
    getName: () => 'UPS',
    isUPSService: () => true,
    isBindingOperationInProgress: () => false,
  };
  it('should render the bind button', () => {
    const wrapper = mount(<BoundServiceRow service={service} />);
    console.log(wrapper.debug());
    expect(wrapper.find('BindButton')).toHaveLength(1);
  });
  it('should render the binding status', () => {
    const wrapper = shallow(<BoundServiceRow service={service} />);
    expect(wrapper.find('BindingStatus')).toHaveLength(1);
  });
});
```
This has the problem we discussed yesterday.

Reply:
@wei-lee @aliok I made your test work here: https://github.com/ziccardi/mobile-developer-console/blob/23b0c219eff1d88eaa28438dd9ece78f82eabf39/ui/src/components/mobileservices/BoundServiceRow.test.js#L47

To be able to access passed in properties, you need to use `mount` since `shallow` only render the result of the `render` method, thus removing the `parent` (the `BoundServiceRow` component) that would contain the parameter.

However, since `mount` renders the whole component, you need to provide a `store` and a `router`.

For this and other reasons, we should avoid using `mount`.

Reply:
I adapted the tests thanks to your suggestion @ziccardi 
https://github.com/aerogear/mobile-developer-console/pull/143/files#diff-2ee813bbe5ee18e680e26b26005a0597R73

Should be good enough now